### PR TITLE
add note on BenchmarkCI.jl to docs

### PR DESCRIPTION
**Pull request by IanButterworth**
Intends to make it a bit easier to find out how to run benchmarks on CI

### CODE REVIEW

**Comment by ianshmean** [comment]
@tkf I see that https://github.com/tkf/BenchmarkCI.jl is still experimental. Should I mark it as experimental in the docs?

File: docs/src/run_benchmarks.md [hunk 1]
```diff
@@ -68,4 +68,11 @@ Benchmarks can be saved and read using `writeresults` and ``readresults` respect
 ```@docs
 PkgBenchmark.readresults
 PkgBenchmark.writeresults
-```
\ No newline at end of file
+```
+
+## Running a benchmark suite in CI
```
Comment (omus) [inline docs/src/run_benchmarks.md:73]: Seems redundant given that the title of this page is "Running a benchmark suite":
```suggestion
## CI Integration
```

**Review by omus — APPROVED** [review]
Otherwise LGTM

**Comment by DilumAluthge** [comment]
Now is probably a good time to move the CI for this repo over to GitHub Actions.

**Pull request merged**